src/codeu/chat/client/simplegui/ChatSimpleGui.java

```diff
@@ -62,56 +62,75 @@ private Border paneBorder() {
   private void initialize() {
 
     // Outermost frame.
-    // NOTE: may have tweak size, or place in scrollable panel.
-    mainFrame = new JFrame("Chat");
+    // NOTE: may have to tweak size, or place in scrollable panel.
+    mainFrame = new JFrame("ChatU");
     mainFrame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
     mainFrame.setSize(790, 450);
 
     // Main View - outermost graphics panel.
     final JPanel mainViewPanel = new JPanel(new GridBagLayout());
-    mainViewPanel.setBorder(paneBorder());
+    //mainViewPanel.setBorder(paneBorder());
```

Comment:
Note: while it's ok to have dead/commented out code in your own branches for testing, prefer not to have it checked into a main shared repo.  This is because usually as APIs and other code changes, the commented out code might no longer make sense or compile or some certain assumptions change and it will now cause tests to fail, etc